core/services/ping/ping1d_driver.py

```diff
@@ -0,0 +1,25 @@
+import logging
+from shutil import which
+from subprocess import Popen
+from typing import Optional
+
+from pingdriver import PingDriver
+from pingutils import PingDeviceDescriptor
+
+
+class Ping1DDriver(PingDriver):
+    def __init__(self, ping: PingDeviceDescriptor, port: int) -> None:
+        super().__init__(ping, port)
+        self.process: Optional[Popen[bytes]] = None
+
+    def start(self) -> None:
+        bridges = which("bridges")
+        command_line = f"{bridges} -u 0.0.0.0:{self.port} -p {self.ping.port.device}:115200"
+        logging.info(f"launching {command_line}")
+        # pylint: disable=consider-using-with
+        self.process = Popen(command_line.split(" "))
+
+    def stop(self) -> None:
+        logging.info(f"Forcing Ping1d at port {self.port} to stop.")
+        if self.process:
+            self.process.kill()
```

Comment:
`__del__` should call stop, otherwise we could have bridges running with a dead python code, resulting in a deadend of the sensor without a system reboot.

Reply:
You should do it in `PingDriver` base class